PhysicsTools/PatAlgos/python/producersLayer1/lowPtElectronProducer_cff.py

```diff
@@ -11,42 +11,65 @@
+    embedGenMatch    = cms.bool(False),
     genParticleMatch = cms.InputTag("lowPtElectronMatch"),
-    # overrides
-    addElectronID = cms.bool(False),
-    addGenMatch = cms.bool(True),
-    addMVAVariables = cms.bool(False),
-    addPFClusterIso = cms.bool(False),
-    electronIDSources = cms.PSet(),
-    computeMiniIso = cms.bool(False),
-    isoDeposits = cms.PSet(),
-    isolationValues = cms.PSet(),
-    isolationValuesNoPFId = cms.PSet(),
-    miniIsoParamsB = cms.vdouble(),
-    miniIsoParamsE = cms.vdouble(),
-    usePfCandidateMultiMap = cms.bool(False),
-    # embedding
-    embedBasicClusters          = cms.bool(False),
-    embedGenMatch               = cms.bool(False),
+
+    # Electron ID
+    addElectronID = cms.bool(True),
+    electronIDSources = cms.PSet(
+        unbiased = cms.InputTag("rekeyLowPtGsfElectronSeedValueMaps:unbiased"),
+        ptbiased = cms.InputTag("rekeyLowPtGsfElectronSeedValueMaps:ptbiased"),
+    ),
+
+    # User data to add
+    userData = cms.PSet(
+        userClasses = cms.PSet( src = cms.VInputTag('') ),
+        userFloats = cms.PSet( src = cms.VInputTag('') ),
+        userInts = cms.PSet( src = cms.VInputTag('') ),
+        userCands = cms.PSet( src = cms.VInputTag('') ),
+        userFunctions = cms.vstring(),
+        userFunctionLabels = cms.vstring()
+    ),
+
+    # Embedding of RECO/AOD items
     embedGsfElectronCore        = cms.bool(False),
-    embedGsfTrack               = cms.bool(False),
-    embedHighLevelSelection     = cms.bool(False),
     embedPFCandidate            = cms.bool(False),
-    embedPflowBasicClusters     = cms.bool(False),
-    embedPflowPreshowerClusters = cms.bool(False),
-    embedPflowSuperCluster      = cms.bool(False),
+    embedGsfTrack               = cms.bool(False),
+    embedTrack                  = cms.bool(True),
+    embedSuperCluster           = cms.bool(False),
+    embedSeedCluster            = cms.bool(False),
+    embedBasicClusters          = cms.bool(False),
     embedPreshowerClusters      = cms.bool(False),
     embedRecHits                = cms.bool(False),
-    embedSeedCluster            = cms.bool(False),
-    embedSuperCluster           = cms.bool(False),
-    embedTrack                  = cms.bool(True),
-    )
+    embedPflowSuperCluster      = cms.bool(False),
+    embedPflowBasicClusters     = cms.bool(False),
+    embedPflowPreshowerClusters = cms.bool(False),
+
+    # Miscellaneous flags
+    useParticleFlow         = cms.bool(False),
+    usePfCandidateMultiMap  = cms.bool(False),
+    addMVAVariables         = cms.bool(False),
+    addEfficiencies         = cms.bool(False),
+    addResolutions          = cms.bool(False),
+    embedHighLevelSelection = cms.bool(False),
+    computeMiniIso          = cms.bool(False),
+    addPFClusterIso         = cms.bool(False),
+    addPuppiIsolation       = cms.bool(False),
+    isoDeposits             = cms.PSet(),
+    userIsolation           = cms.PSet(),
```

Comment:
Removed everything except `userIsolation = cms.PSet()`, which results in a cms::Exception being thrown, presumably because this configurable is updated somehow before I clone the _cfi. (This is one reason why I originally specified everything explicitly in my _cff, even if there is no apparent override).